webhook_to_fedora_messaging/endpoints/message.py

```diff
@@ -0,0 +1,36 @@
+from flask import Blueprint, Flask, request, Response, Request
+from ..database import db
+from ..models.service import Service
+from ..models.user import User
+from sqlalchemy_helpers import get_or_create
+from .util import not_found, success, bad_request, created, conflict, validate_request, create_event, query_fasjson
+from .parser.parser import msg_parser
+from fedora_messaging import api
+
+
+app = Flask(__name__)
+message_endpoint = Blueprint("message_endpoint", __name__)
+
+
+@message_endpoint.route("/message", methods=["POST"])
+def create_msg():
+    """
+    Used for creating a new message by sending a post request to /message path
+        
+        Request Body:
+        service_uuid: Service related to message.    
+        
+    """
+    if not validate_request(request, fields=['service_uuid']):
+        return bad_request()
+    
+    session = db.Session()
+    body = request.json
+    
+    service = session.query(Service).filter(Service.id == body['service_uuid']).first()
```

Comment:
This is the old way of make SQLAlchemy queries. The new style is:
```suggestion
from sqlalchemy import select
from sqlalchemy.exc import NoResultFound
try:
    service = db.session.scalar(select(Service).where(Service.uuid == service_uuid)).one()
except NoResultFound:
    return not_found()
```